### DISCUSSION TITLE
Analytics for my project are not working

### DISCUSSION THREAD

**Discussion by luigicallero** [WalletConnect/walletconnect-monorepo#4379] [Developer Support]
I have already checked the instructions here: https://docs.walletconnect.com/cloud/analytics#web3modal
I do not understand if there are any previous steps. Project is created in walletconnect and I have even deployed it in vercel.
Any missing steps on my side?

**Comment by glitch-txs** [comment]
hi could you share your project id?

**Reply by luigicallero** [reply]
project id: 939e58a1f2d1c4d29ce6ef9e553dd337

**Comment by glitch-txs** [comment]
Currently analytics only shows production interactions
